### DISCUSSION TITLE
osquery and sysmon

### DISCUSSION THREAD

**Discussion by tommorgan365** [Security-Onion-Solutions/securityonion#1757] [Unsupported Versions] [answered]
Reading through the documentation 

Do you have any examples on this one?

Add the new query to a query pack that targets a Windows host - how often it should run depends on log volume on the local host; start off with 180 seconds, differential logging.

**Comment by weslambert** [comment, marked as answer]
It should be something like the following (from the Fleet UI):

Query -> Manage Queries -> Create New Query (Save)
Packs -> Manage Packs -> Select + Edit Pack (Modify Targets for Windows only if needed, Modify Logging options as needed) 
Save pack + Enable pack, if needed.
